pkg/apis/core/v1alpha1/types_shoot.go

```diff
@@ -853,11 +855,19 @@ type Volume struct {
 	Encrypted *bool `json:"encrypted,omitempty"`
 }
 
+// Cri contains information about the Container Runtimes.
+type Cri struct {
+	// Enable Kubelet support for ContainerD
+	EnableContainerD bool `json:"enableContainerD"`
+}
+
 var (
 	// DefaultWorkerMaxSurge is the default value for Worker MaxSurge.
 	DefaultWorkerMaxSurge = intstr.FromInt(1)
 	// DefaultWorkerMaxUnavailable is the default value for Worker MaxUnavailable.
 	DefaultWorkerMaxUnavailable = intstr.FromInt(0)
+	// DefaultCri is the default value for worker Cri.
+	DefaultWorkerCri = Cri{EnableContainerD: true}
```

Comment:
```suggestion
	DefaultWorkerCRI = CRI{Containerd: true}
```

The default should be false, I think

Reply:
After the discussion i will remove all true defaults.
It will be a change triggered only by users and explicitly.
Also updating the GEP.